app/controllers/homepage_controller.rb

```diff
@@ -21,7 +21,9 @@ def index
     listing_shape_menu_enabled = all_shapes.size > 1
     @show_categories = @categories.size > 1
     show_price_filter = @current_community.show_price_filter && all_shapes.any? { |s| s[:price_enabled] }
-    @show_custom_fields = @current_community.custom_fields.any?(&:can_filter?) || show_price_filter
+
+    filters = @current_community.custom_fields.where(search_filter: true).sort
```

Comment:
I'm wondering what kind of SQL query this generates with the sort. What's the sort based on when no field is specified? Is a single field index good enough to satisfy this (if it's e.g. filtering by search_filter but sorting by id)?